openedxscorm/scormxblock.py

```diff
@@ -164,6 +164,12 @@ def scorm_storage(xblock):
         default=False,
         scope=Scope.settings,
     )
+    enable_fullscreen_button = Boolean(
+        display_name=_("Show Fullscreen Button"),
+        help=_("Select True to show fullscreen button in the SCORM content"),
+        default=False,
```

Comment:
@zameel7 could you please set default value of this attribute to `True`

Reply:
Sure!